backend/src/controllers/managerController.ts

```diff
@@ -0,0 +1,55 @@
+import { Request, Response } from 'express';
+import ContactManager from '../models/ContactManager';
+import { generateToken } from '../middleware/tokens';
+
+interface StringBearer {
+    page: string;
```

Comment:
```suggestion
    page?: string;
```
I got a compilation error without this, probably because page could be unspecified in the request